[Bug] Multiple broken paths after folder relocation in kfp v2 documentation

### Search before asking

- [X] I searched the [issues](https://github.com/IBM/data-prep-lab/issues) and found no similar issues.


### Component

Documentation

### What happened + What you expected to happen

Several broken links and wrong path references in Makefile and sample script, for https://github.com/IBM/data-prep-kit/tree/dev/examples/kfp-pipelines/superworkflows/ray/kfp_v2. It seems it came after folder relocation.

### Reproduction script

For broken links, just hit the first link in https://github.com/IBM/data-prep-kit/tree/dev/examples/kfp-pipelines/superworkflows/ray/kfp_v2, which takes you to 404
For the Makefile, just running:
```
Make clean
Makefile:3: [...]/data-prep-kit/examples/kfp-pipelines/superworkflows/ray/kfp_v2/../../../../transforms/.make.workflows: No such file or directory
make: *** No rule to make target `[...]/data-prep-kit/examples/kfp-pipelines/superworkflows/ray/kfp_v2/../../../../transforms/.make.workflows'.  Stop.
```

### Anything else

_No response_

### OS

Other

### Python

Other

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

--- a/examples/kfp-pipelines/superworkflows/ray/kfp_v2/Makefile
+++ b/examples/kfp-pipelines/superworkflows/ray/kfp_v2/Makefile
@@ -1,4 +1,4 @@
-REPOROOT=${CURDIR}/../../../..
+REPOROOT=${CURDIR}/../../../../..
 WORKFLOW_VENV_ACTIVATE=${REPOROOT}/transforms/venv/bin/activate
 include $(REPOROOT)/transforms/.make.workflows
 

--- a/examples/kfp-pipelines/superworkflows/ray/kfp_v2/README.md
+++ b/examples/kfp-pipelines/superworkflows/ray/kfp_v2/README.md
@@ -1,8 +1,8 @@
 # Chaining transforms using KFP V2
 
-As in [super pipelines of KFP v1](../../../doc/multi_transform_pipeline.md), we want to offer an option of running a series of transforms one after the other on the data. But, in KFP v2 we can make it easier to chain transforms using the [nested pipelines](https://www.kubeflow.org/docs/components/pipelines/user-guides/components/compose-components-into-pipelines/#pipelines-as-components) that KFP v2 offers.
+As in [super pipelines of KFP v1](../../../../../kfp/doc/multi_transform_pipeline.md), we want to offer an option of running a series of transforms one after the other on the data. But, in KFP v2 we can make it easier to chain transforms using the [nested pipelines](https://www.kubeflow.org/docs/components/pipelines/user-guides/components/compose-components-into-pipelines/#pipelines-as-components) that KFP v2 offers.
 
-One example of chaining `noop` and `document id` transforms can be found [here](superpipeline_noop_docId_v2.py). When running this pipeline it appears as hierarchical graph with two nested pipelines, one for each transform as shown in the following screenshots.
+One example of chaining `noop` and `document id` transforms can be found [here](superpipeline_noop_docId_v2_wf.py). When running this pipeline it appears as hierarchical graph with two nested pipelines, one for each transform as shown in the following screenshots.
 
 `root` Layer
 ![nested_pipeline](nested_pipeline.png)
@@ -27,6 +27,6 @@ Another useful feature of the KFP v2 is the `Json` editor for the `dict` type in
 cd examples/kfp/superworkflows/ray/kfp_v2/
 make clean
 export KFPv2=1
-export PYTHONPATH=../../../../transforms
+export PYTHONPATH=../../../../../transforms
 make workflow-build
 ```

